LICENSE.rst

```diff
@@ -1,7 +1,7 @@
 License
 =======
 
-Copyright (c) 2018, Transcriptic Inc
+Copyright (c) 2020, Strateos Inc
```

Comment:
Not allowed by the license. To remove the original copyright notice, you need the written consent of the original copyright-holder(s). Otherwise, keep the original copyright notice intact and add your new copyright notice in the line above.  

Alternative:  
Add a description above/below the current License text that explains exactly, which directories are originally Copyright Transcriptic Inc and which directories are originally(!) copyright Strateos Inc and state, that for the files with original Copyright Transcriptic Inc there exists a seperate License. Then Add a copy of the old (unmodified) license file to each of the directories which are from Transcriptic Inc.

Reply:
Hi DerAndre! Thanks for the contribution here.
For context, Transcriptic Inc. has undergone a merger and is now known as Strateos Inc. The purpose of this PR is to reflect this change.

For all intents and purposes, we have the consent of the original copyright holders.